build/configs/artik053/nettest/defconfig

```diff
@@ -1119,35 +1119,11 @@ CONFIG_MEMCPY_INDEXED_COPY=y
 #
 # CURL Options
 #
-CONFIG_ENABLE_CURL=y
+# CONFIG_ENABLE_CURL is not set
```

Comment:
I don't agree disabling CURL. Currently it works but after merging this PR, it might have compilation error. We should resolve error, not disabling.

Reply:
I am sorry, compilation error is not comming from this PR. Anyway we will try to resolve it.